src/folder/folder.validator.ts

```diff
@@ -4,11 +4,11 @@ export class FolderValidator {
   public static compareFolders(folder1: IFolder, folder2: IFolder): boolean {
     return (
       (folder1._id + '' === folder2._id + '') &&
-      folder1.name === folder2.name &&
-      folder1.parent === folder2.parent &&
+      folder1.name + '' === folder2.name + '' &&
```

Comment:
Convert to string via `String()` function and not by adding an empty string.

Reply:
Done